.github/workflows/deploy-prod.yml

```
@@ -0,0 +1,93 @@
+name: Deploy to Production
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+env:
+    ECS_CLUSTER: devops-ic-production-ecs-cluster
+    ECR_REPOSITORY: devops-ic
+    ECS_SERVICE: devops-ic-production-ecs-service
+    ECS_TASK_DEFINITION: devops-ic-production-service
+    CONTAINER_NAME: devops-ic-production
+    REGION: ap-southeast-1
+    NODE_ENV: production
+    APP_ENV: production
+    PORT: 4000
+
+jobs:
+    build:
+        name: Build
+        environment: production
```

what do you think about combining two workflows into one? It will lower the costs for maintaining these workflows, and you will not need to update things twice if something needs to change in the deployment workflow

```suggestion
        environment: ${{ github.ref_name == 'main' && 'production' || 'staging' }}
```

variables like `ECS_*` can be dynamically set based on current env as well

---

Nice suggestion, updated in [8d4c4b5](https://github.com/nvminhtue/tyson_devops_ic_app/pull/11/commits/8d4c4b56ff84ed262371e04cd8b3dbea09fbe0c4)